becquerel/core/fitting.py

```diff
@@ -624,6 +647,116 @@ def fit(self, backend="lmfit"):
                 calc_covar=False,
             )  # no, bounds, default would be L-BFGS-B'
             # TODO: Calculate errors breaks minimization right now
+
+        elif self.backend in ["iminuit", "minuit"]:
+            # Translate a model from lmfit to minuit
+
+            # Poisson loss given the model specified by args
+            def model_loss(*args):
+                # Convert args to kwargs as lmfit.model.eval _requires_ kwargs,
+                # while (I think) the cost function passed to the Minuit object
+                # _requires_ args.
+                # TODO: check if the above is true for Minuit
+                # TODO: this feels dangerous!
+                # TODO: test out-of-order kwargs
+                kwargs = {self.model.param_names[i]: arg for i, arg in enumerate(args)}
+                y_eval = self.model.eval(x=self.x_roi, **kwargs)
+                if self.dx_roi is not None:
+                    y_eval *= self.dx_roi
+                lp = poisson_loss(y_eval, self.y_roi)
+                return lp
```

Comment:
Support least squares loss as well?

Reply:
Just raise `NotImplementedError` for now and note that `lmfit` can handle this case.